cmd/configure.ac

```diff
@@ -209,6 +226,15 @@ AC_ARG_ENABLE([static-libseccomp],
     esac], [enable_static_libseccomp=no])
 AM_CONDITIONAL([STATIC_LIBSECCOMP], [test "x$enable_static_libseccomp" = "xyes"])
 
+AC_ARG_ENABLE([static-libselinux],
```

Comment:
Can you add a comment that explains when we might want to statically link libselinux.

Reply:
Reexec only. We should probably have a similar comment for libapparmor and libseccomp bits.

Reply:
Is it safe to statically link to libselinux?

Reply:
I can't really address this without more hands-on experience. There is a static library in Fedora repositories. Looking at the source code, the library looks more like a wrapper around the kernel interfaces in /sys/fs/selinux. I think we'll only know what it lands and someone starts making use of it.